pkgs/development/libraries/libcsptr/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchFromGitHub, cmake, git }:
+
+stdenv.mkDerivation rec {
+  name = "libcsptr-${version}";
+  version = "2.0.4";
+
+  src = fetchFromGitHub {
+    owner = "Snaipe";
+    repo = "libcsptr";
+    rev = "v${version}";
+    sha256 = "0i1498h2i6zq3fn3zf3iw7glv6brn597165hnibgwccqa8sh3ich";
+  };
+
+  nativeBuildInputs = [ cmake git ];
```

Comment:
Why do you need git as a build input? In my experience this often indicates impurities in the build.

Reply:
removed